Python/wrfrun/fixIM.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/python
+
+# ===============================================================================
+# === Script to fix ERA5-jpeg-compressed-grib2-produced intermediate files,   ===
+# === by fixing the SEAICE missing values (originally +9999, corrected value  ===
+# === -1E30) and start lon and start lat values (values get slightly altered  ===
+# === during making of grib2 files from grib1 files. We replace the altered   ===
+# === values by the original ones).                                           === 
+# ===                                                                         ===
+# === Created on 2022-01-04.                                                  ===
+# ===                                                                         ===
+# === Author: Mani Mahdinia                                                   ===
+# ===============================================================================    
+
+
+# =================================================================================
+# =================================== Imports =====================================
+# =================================================================================
+
+import os
+from sys import argv
+import pygrib
+import numpy as np
+import pywinter.winter as pyw
+
+
+# =================================================================================
+# ================================ Pre-requisites =================================
+# =================================================================================
+
+pfx_f = "FILE-F:"                                                              # Pywinter output IM file prefix.
+fileout = "FILEOUT"                                                            # Output file name ("FILEOUT"'s a convention in pyWPS).
+SEAICE_fill_missing = -1E30                                                    # Value to use to fill SEAICE missing points.
+[stlat_a, stlon_a] = [86.95000457763672,-199.20001220703125]                   # Correct (grib1) IM start lat and start lon. 
```

Comment:
I fixed this issue through the introduction of the environmental variable "REPLACE_IM_STLATSTLON". If this variable is not defined, then we're good because of the small acceptable differences. If this variable IS defined, then the user knows what they are doing.